.github/workflows/python.yml

```diff
@@ -1,5 +1,12 @@
 name: Python CI
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  release:
+    types:
+      - released
```

Comment:
Why add this type?

Reply:
I don't want to trigger the CI on other release events that are not `released` (like draft). We do the same in other repos (like config service) and it's working well

Reply:
I see. I asked because this is not mentioned: it was about not running the CI on PRs twice.

Reply:
You are right, I will mention it on the PR description

Reply:
You are right, maybe release it's not needed for now, I will add it in a new PR if that's required

Reply:
Up to you :) I'm fine approving this as it is. Was just trying to understand the context.

Reply:
Removed for now